Project-Descriptions-and-Plans/CVE20/CVE20-PA-2024.md

```diff
@@ -0,0 +1,311 @@
+
+
+*This section provides the detailed feature list of the project.*
+*If the features are outlined in detail in the PL, or within a User Manual or other document, refer to that document. If not, the feature list should be provided here*
+
+Detailed features are available at https://docs.openhwgroup.org/projects/cve2-user-manual/en/latest/01_specification/index.html
+
+Note however, that the core complex (coreplex) is no longer part of the CV32E20 feature deliverables. 
+
+ 
+## Resources/ Resource Plan
+*Members (organizations) and individuals committed to work on the project should be listed in this section*
+*The following table is provided as an example*
+
+| Organization					| Person							| Project deliverable focus 			|      
+| --------------------    		| --------------------    			| -------------------- 					|
+| OpenHW Staff	    			| Mario Rodriguez					|	Spike, RTL, verification								|
+| OpenHW Staff	    			| Mike Thompson					|	Technical Project Leadership							|
+| OpenHW Staff	    			| Duncan Bees					|	Project management								|
+ 
+*Every project deliverable must have at least one resource assigned at PA*.
+*View of resources needed/available to be added*
+ 
+ 
+ ## Project Leadership Roles
+*The person or persons responsible for project planning/ management should be listed* 
+ - *Project Manager(s)*
+ - *Technical Project Leader(s)*
+ - *Committers on project repos*
+* Often these roles would be taken the same person(s).*
+
+| Role									| Person							| Organization							| Committer Status						|       
+| --------------------    				| --------------------    			| -------------------- 					| -------------------- 					| 
+| Project Manager(s) (if applicable)	| 	Duncan Bees				  		| OpenHW Group							| Committer								| 
+| Technical Project Leader(s)	    	| 	Mike Thompson				  	| OpenHW Group							| Committer								| 
+| Committer1 on project repo			|									|										| Elected								| 
+| Committer2 on project repo			|									|										| Election to be held					| 
+
+
+*There needs to be at least one CORE-V CORES committer already elected who can review and merge pull requests on the project repos. 
+
+
+
+## Work Breakdown Structure
+
+
+*The project work is broken down starting from high level to low level.*
+- *At high level, work can be broken down by technical component, project phase, area of focus or other method.*
+- *At low level, specific tasks are enumerated with estimated time durations (e.g. 1 person for 2 weeks)* 
+- *If appropriate, tasks are assigned to individuals*
+- *For agile projects, tasks can be assigned to individuals during project execution. If so, initial task backlog without assignment to individual can be created.*
+- *Format of the WBS can be github project board, spreadsheet, or other*.
+
+The work breakdown structure at P.A. is listed in the "task list" tab of the spreadsheet https://docs.google.com/spreadsheets/d/1L4QS830qSwUS_B30AWsCoOTGSXbFLExG/edit#gid=141038687
```

Comment:
Another link that will break.

Reply:
removed and replaced with a reference to the project board